MetaMask/metamask-mobile
# test(3615): CLOSED PR, additional e2e scenarios editing permissions and non permitted networks

## **Description**
Covers with e2e the scenarios:
1. should update chain permissions by granting and revoking network permissions simultaneously
2. should allow switching to permitted network when attempting to use non-permitted network 




## **Related issues**

Contributes to solve issue: https://github.com/MetaMask/MetaMask-planning/issues/2796

## **Manual testing steps**

I have added screenshot below to see how it looks.

1. `yarn watch:clean` 
2. `yarn test:e2e:ios:debug:build`
3. `yarn test:e2e:ios:debug:run <path to test file>`



## **Screenshots/Recordings**



![2024-12-03 12 50 35](https://github.com/user-attachments/assets/634d72f9-6518-446d-941f-9a9b0d99083b)



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
